Steps/Steps.module.less

```diff
@@ -16,14 +16,14 @@
 			font-weight: @sand-steps-step-number-font-weight;
 		}
 
-		&.dots {
-			font-size: @sand-steps-step-dot-font-size;
+		&.highlightCurrent {
+			font-size: @sand-steps-step-highlightCurrent-font-size;
 			opacity: @sand-steps-step-past-opacity;
 			margin: 0;
-			width: @sand-steps-step-width;
+			width: @sand-steps-step-highlightCurrent-width;
```

Comment:
Do we need this too?

Reply:
I added this prop because of the gap between each step circle, but I think it could be handled with just the margin. I modified the margin values.